src/test/java/plugins/MatrixPluginTest.java

```diff
@@ -131,19 +186,89 @@ public void run_configurations_on_with_a_given_label() throws Exception {
         b.getConfiguration("label=label1").shouldContainsConsoleOutput("(Building|Building remotely) on " + s.getName());
     }
 
-    private void assertThatBuildHasRunSequentially(MatrixBuild b) {
-        List<MatrixRun> builds = b.getConfigurations();
+    @Test
+    public void run_extended_test() {
+        job.configure();
+        // Create a [3,3] matrix
+        job.addUserAxis(AXIS_X, AXIS_X_VALUES);
+        job.addUserAxis(AXIS_Y, AXIS_Y_VALUES);
 
-        while (b.isInProgress()) {
-            int running = 0;
-            for (MatrixRun r : builds) {
-                if (r.isInProgress()) {
-                    running++;
+        // Table to Divs: https://www.jenkins.io/doc/developer/views/table-to-div-migration/
+        if (jenkins.getVersion().isOlderThan(new VersionNumber("2.264"))) {
```

Comment:
not needed - the only supported versions here lie between the current LTS and the latest weekly

Reply:
removed in 9516b55693bd9a611f9d36e88e9e70dc11daef8d